### DISCUSSION TITLE
For the Creators/Contributors of this repo

### DISCUSSION THREAD

**Discussion by HorrorBest** [electronstudio/raylib-python-cffi#37] [General]
Please mention In the Github description or where the installation guide is that The functions have been changed from 
initwindow() in "C" to init_window() in "python" or make your own cheatsheet ( I can do It if you want me to ) Because For a moment I thought It was broken or missing functions

**Comment by electronstudio** [comment]
The function names in raylib.static are identical to the C functions. Names have not been changed. 

The functions names raylib.pyray are the same as raylib.static but have been changed to snake case. The readme already says this. 

There is a Sphinx generated API doc that lists them somewhere too. You can contribute additional docs of course but if you write a cheat sheet by hand you would have to update it every time raylib changes their API, which is often, and is why I haven’t done it myself.

**Reply by HorrorBest** [reply]
maybe using the word "changed" is wrong what I mean is that they have the "_" thats why cause I was so confused on why they didn't work.
